:bug: Fix date-related objects in containers not being serialized

viktorvanwijk opened the pull request:
We can use the DjangoJSONEncoder for this

viktorvanwijk committed:
:bug: Fix date-related objects in containers not being serialized

We can use the DjangoJSONEncoder for this

sergei-maertens reviewed (approved):
Can you take a look at why CI is failing?

viktorvanwijk committed:
:green_heart: Fix CI

Docker compose (``docker-compose``) was deprecated a long time ago:
https://github.com/actions/runner-images/issues/9692
